croptalk/load_data.py

```diff
@@ -12,16 +13,8 @@
 # Create the SQLAlchemy engine
 engine = create_engine(db_url)
 
-# Define your SQL query
-sql_query = """
-SELECT * 
-FROM sob_all_years
-WHERE commodity_year >= 2010
-"""
-
-# Execute the SQL query
-with engine.connect() as connection:
-    result = connection.execute(text(sql_query))
-    rows = result.fetchall()
-
-SOB = pd.DataFrame(data=rows)
+SOB = query_sql_data(query="""
```

Comment:
I think there will only be these 5 lines + 1 import left in this file/module... I'll let you decide if you still want to keep it here or move it where it is used